app/models/project.rb

```diff
@@ -623,7 +620,7 @@ def check_status(deprecated_or_removed = false)
         update_attribute(:status, nil)
         update_attribute(:deprecation_reason, nil)
       end
-    elsif deprecated_or_removed # if package is already deprecated or removed and isn't anymore, remove status
+    elsif status == "Removed"
```

Comment:
maybe this is actually just `else` ? I think changing nil to nil should automatically do nothing, so is harmless.

Reply:
I think you're right, changed to remove the conditional in [2c1b880](https://github.com/librariesio/libraries.io/pull/3114/commits/2c1b88039e2ea5ebb932cf6ed346fe63b886910e).

Reply:
The condition I was thinking of for this was if the status was currently something like "Help Wanted", the else statement would catch that and change the status to nil.